### PR TITLE
Add unit tests for com.sia.hunter.helper.OnlineTaskHelper

### PR DESCRIPTION
**Pull request by shafi-1**
I've analysed your codebase and noticed that `com.sia.hunter.helper.OnlineTaskHelper` is not fully tested. I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important in a subsequent PR.

### CODE REVIEW

File: sia-task-hunter/src/main/test/com/sia/hunter/OnlineTaskHelperTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  @Test
+  public void testToList() {
+    Assert.assertEquals(Arrays.asList("-100"),
+            OnlineTaskHelper.toList(new Object[]{-100}));
+    Assert.assertEquals(new LinkedList<String>(),
+            OnlineTaskHelper.toList(new Object[]{}));
+    Assert.assertEquals(new LinkedList<String>(),
+            OnlineTaskHelper.toList(null));
+  }
+}
```
Comment (JohnLBergqvist) [inline sia-task-hunter/src/main/test/com/sia/hunter/OnlineTaskHelperTest.java]: Need to add a new line at the end of the file.

**Review by JohnLBergqvist — CHANGES_REQUESTED** [review]

File: sia-task-hunter/pom.xml [hunk 2]
```diff
@@ -86,6 +86,41 @@
...
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.8.2</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>com.diffblue</groupId>
```
Comment (ThomasPerkins1123) [inline sia-task-hunter/pom.xml]: This dependency should not be needed

File: sia-task-hunter/src/main/test/com/sia/hunter/OnlineTaskHelperTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.Arrays;
+import java.util.LinkedList;
+
+@RunWith(PowerMockRunner.class)
+public class OnlineTaskHelperTest {
+
+  @PrepareForTest(RequestMethod.class)
+  @Test
+  public void testCheckRequestMethod() {
+    final RequestMethod requestMethod1 = PowerMockito.mock(RequestMethod.class);
```
Comment (ThomasPerkins1123) [inline sia-task-hunter/src/main/test/com/sia/hunter/OnlineTaskHelperTest.java:21]: Does this actually need to be mocked?

**Review by paulkennethkent — APPROVED** [review]

File: sia-task-hunter/pom.xml [hunk 4]
```diff
@@ -86,6 +86,41 @@
...
+
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-module-junit4</artifactId>
+			<version>1.6.5</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.mockito</groupId>
```
Comment (ThomasPerkins1123) [inline sia-task-hunter/pom.xml:104]: I don't think you are using mockito either but I may be wrong

**Review by ThomasPerkins1123 — CHANGES_REQUESTED** [review]

File: sia-task-hunter/src/main/test/com/sia/hunter/OnlineTaskHelperTest.java (hunk 3 above)
Comment (shafi-1) [reply sia-task-hunter/src/main/test/com/sia/hunter/OnlineTaskHelperTest.java:21]: Yes it does

File: sia-task-hunter/pom.xml (hunk 4 above)
Comment (shafi-1) [reply sia-task-hunter/pom.xml:104]: Yes we are using mockito

File: sia-task-hunter/pom.xml (hunk 2 above)
Comment (shafi-1) [reply sia-task-hunter/pom.xml]: removed

**Review by ThomasPerkins1123 — APPROVED** [review]
I'll approve but just remember i'm not a senior review so you still need to wait for a review from John

**Review by JohnLBergqvist — APPROVED** [review]

**Pull request merged**